app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php

```diff
@@ -39,7 +39,11 @@
         </x-filament::badge>
     @endif
     @if ($getRecord()->public && !empty($getRecord()->category_id))
+        <x-filament::badge>
+            Public
+        </x-filament::badge>
         <button
+            class="flex items-center justify-center"
```

Comment:
This doesn't need to be a button and shouldn't so that we can get rid of the button style and border around the icon. The copy on click functionality does not come from it being a button. It comes from the `x-on:click`. You can just get rid of the button HTML and put the onclick on the x-filament::badge directly

Reply:
Done